src/components/Account/AccountModule.js

```diff
@@ -222,10 +272,21 @@ const AccountModule = ({ compact }) => {
                 `}
               >
                 {(() => {
+                  if (screen.id === 'connecting-network') {
+                    return (
+                      <ScreenPromptingAction
+                        actionTitle={actionTitle}
+                        actionDescription={actionDescription}
+                        logo={xDaiLogo}
```

Comment:
I'd set the logo as a property in the network object in `networks.js` instead of hard-coding it here